crates/router/src/core/payments/operations/payment_update.rs

```diff
@@ -414,6 +422,8 @@ impl<F: Clone> UpdateTracker<F, PaymentData<F>, api::PaymentsRequest> for Paymen
 
         let return_url = payment_data.payment_intent.return_url.clone();
         let setup_future_usage = payment_data.payment_intent.setup_future_usage;
+        let business_label = Some(payment_data.payment_intent.business_label.clone());
+        let business_country = Some(payment_data.payment_intent.business_country);
 
         payment_data.payment_intent = db
             .update_payment_intent(
```

Comment:
We should also update payment attempt with these details right?

Reply:
Yes, payment attempt should be updated with business_sub_label